mod/web_services/classes/ApiRegistry.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * API method registry
+ */
+class ApiRegistry {
+
+	const API_VERSION = '2.0';
+	
+	static $methods = array();
```

Comment:
Make this whole class dynamic and make _elgg_ws_registry() return a singleton instance.